atlas_admin/helpers.py

```diff
@@ -387,29 +386,17 @@ def statBreakdown():
 def sitesByStat(themeName=None):
     """
     Return a list of sites with the requested statistic
+    Displays on individual Stat List page /instances/th/<themeName> using instances/sitestats.html
     """
     if themeName:
-        q = get_internal('statistics',  **{"variable_theme_default": themeName})
-    else:
-        q = get_internal('statistics')
-    res = q[0] if len(q) > 0 else {}
-    totalItems = res.get('_meta', None).get('total', None)
+        findThisElement = {"variable_theme_default": themeName}
+
+    results, totalItems = getAllResults(atlasType='statistics', **findThisElement)
 
-    # If more items exist than initial request, reset max_results to total to get a full export
-    if totalItems > res.get('_meta', None).get('max_results', None):
-        setArgs = request.args.copy()
-        setArgs['max_results'] = totalItems
-        request.args = setArgs
-        if themeName:
-            qAll = get_internal('statistics',  **{"variable_theme_default": themeName})
-        else:
-            qAll = get_internal('statistics')
-        results = qAll[0].get('_items', None)
-    else:
-        results = res.get('_items', None)
     # Get list of all instances
+    myList = []
```

Comment:
Can you come up with a more descriptive variable name?

Reply:
unsortedList